src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Color[] color = Color.values();
```

Comment:
use plural for array name
```suggestion
    private Color[] colors = Color.values();
```